app-core/src/main/java/pm/gnosis/heimdall/util/ContextUtils.kt

```diff
@@ -35,3 +37,13 @@ fun Context.shareExternalText(text: String, dialogTitle: String = "") {
 
 fun Activity.scanQrCode() = ZxingIntentIntegrator(this).initiateScan(ZxingIntentIntegrator.QR_CODE_TYPES)
 fun Fragment.scanQrCode() = ZxingIntentIntegrator(this).initiateScan(ZxingIntentIntegrator.QR_CODE_TYPES)
+
+fun <T : AppCompatActivity> Activity.startActivity(activity: KClass<T>, noHistory: Boolean = false) {
+    val i = Intent(this, activity.java)
```

Comment:
I would propose to pass in an Intent, else it is not possible to add params with this method. Also I like the pattern that the activity that is started has an method to create the intent. This makes sure that you don't forget expected parameters